### PR TITLE
si404 USA fixes & IN changes

### PR DESCRIPTION
**Pull request by si404**
Fixing stuff with US changes from last submission and making the following noteworthy changes in Indiana:

15-08-20;(USA) Indiana;I-164;in.i164;Route truncated at north end from exit 20 to US41
15-08-20;(USA) Indiana;Veterans Memorial Parkway;in.vetmempkwy;Route added
15-08-20;(USA) Indiana;US52;in.us052;Route removed from Sagamore Parkway, Lafayette and relocated onto US231, Old US231 and Teal Road between western junction with Sagamore Pkwy (formerly southern junction with US231) and eastern junction with Sagamore Pkwy (formerly junction with IN25)

### CODE REVIEW

**Comment by si404** [comment]
Additional updates, having brought Greece up to date (and some other changes).

15-08-20;France;A344;fra.a344;New Route (formerly A444)
15-08-20;France;A350;;Route deleted
15-08-20;France;A444;;Route renamed A344
15-08-20;Greece;A1 (Athens);grc.a001ath;Route extended at north end from exit 43 to PE01
15-08-20;Greece;A1 (Athens);grc.a001ath;Route extended at south end from exit 82 to exit 87
15-08-20;Greece;A1 (Larisa);;Route merged into Athens section
15-08-20;Greece;A1 (Paraskevi);;Route merged into Athens section
15-08-20;Greece;A1 (Kato Egani);grc.a001kat;New Route
15-08-20;Greece;A1 (Thessaloniki);grc.a001the;New Route
15-08-20;Greece;A4;grc.a004;New Route
15-08-20;Greece;A5 (Patras);grc.a005pat;Route extended at north end from exit K1 to exit 3
15-08-20;Greece;A5 (Kaliva);grc.a005kal;New Route
15-08-20;Greece;A7;grc.a007;Route extended at south end from exit 14 to exit 18
15-08-20;Greece;A7 (Thouria);;Route merged into main A7
15-08-20;Greece;A8;grc.a008;Route extended at east end from A6 to EO8
15-08-20;Greece;A8 (Patras);grc.a008pat;New Route
15-08-20;Greece;A11;grc.a011;Route extended at north end from EP34 to EO44a
15-08-20;Greece;A12;grc.a012;New Route
15-08-20;Greece;A23;grc.a023;New Route
15-08-20;Greece;A25;grc.a025;Route extended at north end from Rizana to eastern junction with EO12
15-08-20;Greece;A25;grc.a025;Route extended at south end from exit 24 of the A2 to the junction with EO16g at Nea Moudania
15-08-20;Greece;A25 (Promahonas);grc.a025pro;Route merged with Sidirokastro route
15-08-20;Greece;A25 (Paleokastro);grc.a025pal;Route merged with Sidirokastro route
15-08-20;Greece;A25 (Sidirokastro);grc.a025sid;New Route
15-08-20;Greece;A27;grc.a027;Route truncated at north end from Odos Oik to EO3
15-08-20;Greece;A29;grc.a029;Route extended at north end from exit 3 to exit 7
15-08-20;Greece;A29 (Argos Orestiko);;Route route merged into main A29
15-08-20;Greece;A62;grc.a062;Route extended at north end from Aerolimenas Eleftherios Venizelos to Exhibition Centre
15-08-20;Greece;A62;grc.a062;Route extended at south end from exit K2 to exit K3
15-08-20;Greece;A64;grc.a064;Route extended at east end from exit Y8 to exit Y9
15-08-20;Greece;A90 (Heraklion);grc.a090ira;Route extended at east end from Leo Sen Geo to EO90_E
15-08-20;Greece;A123;grc.a123;New Route
15-08-20;Greece;A581;grc.a581;New Route
15-08-20;Greece;E75 (Thessaloniki);grc.e75the;New Route
15-08-20;Greece;E79;grc.e79;Route extended at south end from A2 exit 23 to EO111
15-08-20;Greece;E94;grc.e94;Route truncated at west end from A8 exit 16 to A8 exit 15
15-08-20;Greece;E94;grc.e94;Route extended at east end from A6 junction 8 to the EO89
15-08-20;Greece;E962;grc.e962;Route extended at south end from eastern junction with the A6 to western junction with the A6
15-08-20;(UK) England;A5758;eng.a5758;New Route

**Comment by si404** [comment]
And Slovakian updates

15-08-20;Slovakia;D1 (Dri);svk.d001dri;Route extended at north end from exit 407 to I68
15-08-20;Slovakia;D1 (Vrutky);svk.d001vru;New Route
15-08-20;Slovakia;D4;svk.d004;Route extended at east end from D2 to Jantarova
15-08-20;Slovakia;D4 (Stupava);svk.d004stu;New Route
15-08-20;Slovakia;R2 (Zvolen);svk.r002zvo;New Route
15-08-20;Slovakia;R2 (Kosice);svk.r002kos;New Route
15-08-20;Slovakia;R3 (Vrutky);svk.r003vru;New Route
15-08-20;Slovakia;R6;svk.r006;Route truncated at north end from II507 to I49
15-08-20;Slovakia;E50;svk.e50;Route removed from I50 and relocated onto D1 between exit 212 of D1 and exit 229

**Comment by jteresco** [comment]
Would you like me to pull these in tonight or wait for additional items to go in?

**Comment by si404** [comment]
Pull them in tonight.

**Comment by jteresco** [comment]
This is a big one, going to try running an update with these now.

File: hwy_data/CZE/czer/cze.r035olo.wpt [hunk 1]
```diff
@@ -9,4 +9,4 @@ I35 http://www.openstreetmap.org/?lat=49.604202&lon=17.218752
 +X202 http://www.openstreetmap.org/?lat=49.581169&lon=17.460022
 290 http://www.openstreetmap.org/?lat=49.568187&lon=17.495427
 +X203 http://www.openstreetmap.org/?lat=49.550997&lon=17.534866
-296 D1 http://www.openstreetmap.org/?lat=49.547224&lon=17.585249
+296 D1 http://www.openstreetmap.org/?lat=49.547224&lon=17.585249
```
Comment (rickmastfan67) [inline hwy_data/CZE/czer/cze.r035olo.wpt]: One minor bug here.  The 'old' label doesn't have the '+' to mark it as hidden.

**Comment by rickmastfan67** [comment]
si, appreciate you fixing the 'hidden' bugs for the US Highways. :)

**Comment by jteresco** [comment]
Fixing many _con.csv errors, but not a big deal.

First showstopper: missing fra.a344.wpt

**Comment by jteresco** [comment]
Also missing: co.co018.wpt

**Comment by jteresco** [comment]
Also missing: wy.seentrd.wpt

**Comment by jteresco** [comment]
Also missing: wy.wy014.wpt

**Comment by jteresco** [comment]
Removing CO 58 and IN 912 from the in-dev state system .csv files as they are currently in usansf.  (Same thing I did for NM 599).  My script, unlike Tim's, requires any file "root" to be unique across all systems, both active and inactive

**Comment by jteresco** [comment]
BTW, the csv entries corresponding to the missing .wpt files are temporarily being removed so I can continue processing.  I will commit/push the fixed/updated .csv files.

**Comment by jteresco** [comment]
I see fra.a344.wpt was pushed to the master, so I added it back to the .csv files before running an update this morning.

**Pull request merged**